Makefile.org

```diff
@@ -424,6 +424,14 @@ clean:	libclean
 	rm -fr $$i/*; \
 	done
 
+distclean: clean
+	-$(RM) `find . -type l -a \! -path "./.git/*"`
+	$(RM) apps/CA.pl
+	$(RM) test/evptests.txt test/newkey.pem test/testkey.pem test/testreq.pem
+	$(RM) tools/c_rehash
+	$(RM) crypto/opensslconf.h
+	$(RM) Makefile Makefile.bak
```

Comment:
Generally speaking (and I know it's a bit more to type), it would be preferable if you had this here:

``` make
distclean: clean
￼	$(RM) Makefile Makefile.bak
￼	@set -e; target=distclean; $(RECURSIVE_BUILD_CMD)
```

and the other removals appropriately spread out among the sub-`Makefile`s.  And yes, it means that you have to add empty `distclean:` rules in a bunch of `Makefile`s.  It *also* means that you won't have to deal with a possible `.git` directory ;-)